database-serviceui/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { TestBed, async } from '@angular/core/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        AppComponent
+      ],
+    }).compileComponents();
+  }));
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title 'database-serviceui'`, () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app.title).toEqual('database-serviceui');
+  });
```

Comment:
ERROR in src/app/app.component.spec.ts:22:16 - error TS2339: Property 'title' does not exist on type 'AppComponent'.

test sind gut, aber bitte passt sie soweit an, dass sie ausführbar sind.